src/components/v2/V2Project/ArchiveV2Project.tsx

```diff
@@ -0,0 +1,144 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, notification } from 'antd'
+import axios from 'axios'
+
+import { NetworkContext } from 'contexts/networkContext'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import { TransactorInstance } from 'hooks/Transactor'
+import { useContext, useState } from 'react'
+import { uploadProjectMetadata } from 'utils/ipfs'
+
+import { readNetwork } from 'constants/networks'
+
+const emitErrorNotification = (message: string) => {
+  notification.error({
+    key: new Date().valueOf().toString(),
+    message,
+    duration: 0,
+  })
+}
+
+export default function ArchiveV2Project({
+  editV2ProjectDetailsTx,
+}: {
+  editV2ProjectDetailsTx: TransactorInstance<{
+    cid: string
+  }>
+}) {
+  const { projectMetadata, projectId, projectOwnerAddress } =
+    useContext(V2ProjectContext)
+  const { userAddress } = useContext(NetworkContext)
+
+  const [isLoadingArchive, setIsLoadingArchive] = useState<boolean>(false)
+
+  const setArchived = async (archived: boolean) => {
+    if (
+      !userAddress ||
+      userAddress.toLowerCase() !== projectOwnerAddress?.toLowerCase()
+    ) {
+      return emitErrorNotification(t`Connected wallet not authorized`)
+    }
+
+    setIsLoadingArchive(true)
+    const uploadedMetadata = await uploadProjectMetadata({
+      ...projectMetadata,
+      archived,
+    })
+    if (!uploadedMetadata.IpfsHash) {
+      return emitErrorNotification(t`Failed to update project metadata`)
+    }
+
+    // Create github issue when archive is requested
+    // https://docs.github.com/en/rest/reference/issues#create-an-issue
+    // Do this first, in case the user closes the page before the on-chain tx completes
+    axios.post(
+      'https://api.github.com/repos/jbx-protocol/juice-interface/issues',
+      {
+        title: `[${archived ? 'ARCHIVE' : 'UNARCHIVE'}] Project: "${
+          projectMetadata?.name
+        }"`,
+        body: `<b>Chain:</b> ${readNetwork.name} \n <b>Project ID:</b> ${projectId} \n`,
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${process.env.REACT_APP_GITHUB_ACCESS_TOKEN}`,
+        },
+      },
+    )
+
+    const tx = await editV2ProjectDetailsTx(
+      { cid: uploadedMetadata.IpfsHash },
+      {
+        onConfirmed: () => {
+          setIsLoadingArchive(false)
+          window.location.reload()
+        },
+      },
+    )
+    if (!tx) {
+      setIsLoadingArchive(false)
```

Comment:
**suggestion** I think we can emit an error noti here.

Reply:
yep - only returns false on error

Reply:
Have added now 🙌